tests/microbenchmarks/all_reduce_rmsnorm_cute_bf16.py

```
@@ -0,0 +1,819 @@
+        best_tpc = tpc_candidates[0]
+        cute_us = float("nan")
+        cute_graph_us = float("inf")
+
+        if not graph_only:
+            fused_fns = {
+                tpc: cute.compile(launch_fused_ar_rmsnorm_bf16, *compile_args, tpc)
+                for tpc in tpc_candidates
+            }
+            cute_us = float("inf")
+            for tpc, fn in fused_fns.items():
+                step = lambda fn=fn: fn(*runtime_args)
```

Ruff (0.15.13)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `step` as a `def`

(E731)